reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/main.py

```diff
@@ -167,6 +169,13 @@ def _model_upload(mclient, args):
         model_upload_parser.add_argument('name', action='store', help="Model name to use")
         model_upload_parser.add_argument('format', action='store', help="Model format to use")
 
+    def _add_action_log_commands(self):
```

Comment:
Add an example(s) usage to examples in the main

Reply:
Fixed